.github/actions/collect-metrics/collect_leaderboard_metrics.py

```
@@ -0,0 +1,319 @@
+
+def collect_reviewers(repo: str, pr_number: str, email_map: dict[str, str]) -> list[str]:
+    result = subprocess.run(
+        [
+            "gh", "api", f"repos/{repo}/pulls/{pr_number}/reviews",
+            "--paginate",
+            "--jq", '[.[] | select(.state=="APPROVED" or .state=="COMMENTED" or .state=="CHANGES_REQUESTED") | select(.user != null) | .user.login | select(endswith("[bot]") | not)] | unique | .[]',
+        ],
+        capture_output=True, text=True, check=True,
+    )
+    # --paginate applies the jq filter per page, so dedupe across pages here.
+    logins = list(dict.fromkeys(l.strip() for l in result.stdout.strip().splitlines() if l.strip()))
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)